.github/actions/next-integration-stat/src/index.ts

```diff
@@ -101,7 +42,6 @@ async function fetchJobLogsFromWorkflow(
   const jobLogsResponse = await fetch(jobLogRedirectResponse.url, {
     headers: {
       Accept: 'application/vnd.github.v3+json',
-      //Authorization: `token ${token}`,
```

Comment:
Adding this token actually causes an HTTP 403. According to the documentation for octokit this URL is publicly accessible for 1 minute, so it shouldn't need a token.